internal/io/csv.go

```diff
@@ -151,6 +162,48 @@ func resizeColBytes(bytes [][]byte, currentRowCount, sizeHint int) {
 	}
 }
 
+// Handle DuplicateColumnNames
+func AddColumnIndexToDuplicateColumns(headers []string) []string {
+	headersMap := make(map[string]int)
+	// Loop through all columsn and find the duplicates.
+	for i, h := range headers {
+		_, ok := headersMap[h]
+		if !ok {
+			headersMap[h] = i
+		}
+	}
+	// For go through all if there is duplicate down the line add index
```

Comment:
Consider rewriting this comment to better reflect why this is done or removing it completely.